internal/cb/errors.go

```diff
@@ -40,3 +51,7 @@ func (e NonRetryableError) Is(target error) bool {
 	_, ok := target.(NonRetryableError)
 	return ok
 }
+
+func (e NonRetryableError) Unwrap() error {
```

Comment:
Would you make the receiver names same?

Reply:
Updated at: https://github.com/hazelcast/hazelcast-go-client/pull/967/commits/478e72045d6bbc797e40b496ac92fc1bea5a6d48